potku.py

```diff
@@ -1485,6 +1486,28 @@ def __open_manual(self):
                 self, "Not found",
                 "There is no manual to be found!",
                 QtWidgets.QMessageBox.Ok, QtWidgets.QMessageBox.Ok)
+                
+                
+    def __open_data_help(self):
+        """Open data help file.
+        """
+        # TODO duplicate code with __open_manual
```

Comment:
Yup. Refactor the duplicated code into a single function that takes a file path to be opened as an argument.